server/app.py

```
@@ -1484,45 +1483,14 @@ async def get_user_scores(request: Request):
-
-                end_time = time.time()
                 logger.info(
-                    f"Time taken to calculate personal ranking: {end_time - start_time:.2f} seconds"
+                    "User scores endpoint temporarily disabled; skipping Firebase fetch for user %s",
+                    user_id,
                 )
 
+                scores_over_time: list = []
                 response = JSONResponse(content=scores_over_time)
+                response.headers["x-leaderboard-disabled"] = "true"
                 return response
```

Ruff (0.13.3)

Consider moving this statement to an `else` block

(TRY300)